### PR TITLE
Page PDF feature

### PR DESCRIPTION
**Pull request by cbutcosk**
This PR updates the PDF transform, config options, and CLI for creating individual PDFs from pages in a quire publication.

This PR replaces #910 but incorporates review items from that thread. It also depends on https://github.com/thegetty/quire-starter-default/pull/43 , already merged.

### CODE REVIEW

**Comment by cbutcosk** [comment]
TODO before release:
- [ ] Refine TOC item PDF link styles + logic
- [ ] Use `@download` attrib on tombstone PDF button
- [ ] Refactor `pageHeader` to remove manual arg passing for PDF link
- [ ] Check nested CSS compilation warnings on PDF asset bundle run
- [ ] Tests

**Comment by cbutcosk** [comment]
For styles, etc testing, please use the styles + config.yaml in this zip:
[qsd.zip](https://github.com/thegetty/quire/files/14981324/qsd.zip)

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -1,6 +1,8 @@
 Thank you for contributing to Quire! Please complete the form below to submit your pull request for review. 
 
-*For the Title of this pull request, please use the format "Type/Issue-#: Brief description." For Type, the options are Fix, Feature, Docs, or Chore. Issue-# is only needed if this pull request addresses an [exisiting issue](https://github.com/thegetty/quire/issues).*
+*For the Title of this pull request, please use the format "Type/Issue-#: Brief description." For Type, the options are Fix, Feature, Docs, or Chore. Issue-# is only needed if this pull request addresses an [existing issue](https://github.com/thegetty/quire/issues).*
+
+**Before submitting your PR, make sure that you can run the following commands: `quire preview`, `quire build`, `quire pdf`, and `quire epub`. If any of those core quire commands throw errors/yield unexpected behavior, we will NOT review the PR!**
```
Comment (mphstudios) [inline .github/pull_request_template.md:5]: @cbutcosk let's pull this out and discuss with @Erin-Cecele updating the pull-request template separately.

File: packages/cli/src/lib/pdf/split.js [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+    const sectionFn = `${filename}-${sectionId}.pdf`
+    const sectionFp = path.join( paths.output, outputDir, sectionFn )
+
+    resultFiles[sectionFp] = section
+
+  }
+
+  return resultFiles
+
+}
```
Comment (mphstudios) [inline packages/cli/src/lib/pdf/split.js:62]:
```suggestion

    resultFiles[sectionFp] = section
  }

  return resultFiles
}
```

**Review by mphstudios — APPROVED** [review]
@cbutcosk this looks great.

**Review by mphstudios — DISMISSED** [review]

**Review by mphstudios — APPROVED** [review]

**Pull request merged**